ui/app/index.html

```diff
@@ -1,32 +1,32 @@
 <!DOCTYPE html>
 <html lang="en" ng-app="am">
 <head>
-	<script src="/lib/jquery.min.js"></script>
```

Comment:
Not a fan of relative locations. This makes it a hard requirement that all entrypoints are mounted under the root (like `/silences/elapsed` would not be possible to add).

Alertmanager unfortunatley uses the awful angular path syntax though, so the requirement is already fulfilled.